Mac.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Hi @fengga - looks like we figured out a new way of doing this. It's called solution filters. See https://learn.microsoft.com/en-us/visualstudio/msbuild/solution-filters?view=vs-2022 

There's already an example https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/LibsAndSamples.sdk.slnf

It would be better to just add a `LibsAndSamples.MacDesktop.slnf` or smth like that?

Reply:
Thanks, @bgavrilMS ! 
The mac maui sample app can be directed added into LibsAndSamples.sln